docs/user-manual/en/using-jms.md

```diff
@@ -124,7 +124,7 @@ factory like so, `tcp://remote-host:5445?ssl-enabled=true`.
 
 All the properties available for the `tcp` scheme are described in [the
 documentation regarding the Netty
-transport](#configuring-transports.netty).
```

Comment:
The proper way to link would be  #configuring-the-netty-transport

no need for the <a link>